doc/sphinx/source/recipes/recipe_oceans.rst

```diff
@@ -42,6 +42,7 @@ Available recipes
 * recipe_ocean_scalar_fields.yml_
 * recipe_ocean_bgc.yml_
 * recipe_ocean_quadmap.yml_
+* recipe_ocean_Landschutzer2014.yml_
```

Comment:
Please put recipes in alphabetical order

Reply:
@bouweandela I think it is better to revise the entire `recipe_oceans.rst` in a separate PR, as also the paragraphs of each recipe should be reordered to be consistent.